<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addNews</class>
 <widget class="QWidget" name="addNews">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>53</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>黑体</family>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="SearchLineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>251</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="searchPushButton">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>10</y>
     <width>81</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>搜索</string>
   </property>
  </widget>
  <widget class="QLabel" name="headLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>73</y>
     <width>60</width>
     <height>55</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>70</y>
     <width>91</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="signalLabel">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>103</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="vTitleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>221</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>验证信息</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="verifyTextEdit">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>331</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="remarkLineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>111</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="rTitleLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>270</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>备注</string>
   </property>
  </widget>
  <widget class="QLabel" name="gTitleLabel">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>270</y>
     <width>41</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>分组</string>
   </property>
  </widget>
  <widget class="QComboBox" name="groupComboBox">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>270</y>
     <width>131</width>
     <height>27</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>我的好友</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="sendPushButton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>310</y>
     <width>331</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>发送申请</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
